src/core_modules/capture-core/components/WidgetEventEdit/WidgetEventEdit.container.tsx

```diff
@@ -89,14 +90,13 @@ const WidgetEventEditPlain = ({
     onDeleteEvent,
     onDeleteEventRelationship,
     classes,
-}: PlainProps) => {
+}: Props) => {
     useEffect(() => inMemoryFileStore.clear, []);
 
     const supportsChangelog = useFeature(FEATURES.changelogs);
     const { currentPageMode } = useEnrollmentEditEventPageMode(eventStatus);
     const [changeLogIsOpen, setChangeLogIsOpen] = useState(false);
-    // "Edit event"-button depends on loadedValues. Delay rendering component until loadedValues has been initialized.
```

Comment:
Put back the comment. Do not delete comments and focus only on the he Flow to Typescript migration